tii_zcu111.py

```diff
@@ -0,0 +1,92 @@
+import pathlib
+
+from qibolab.channels import Channel, ChannelMap
+from qibolab.instruments.erasynth import ERA
+from qibolab.instruments.rfsoc import RFSoC as TII_ZCU111
+from qibolab.platform import Platform
+
+NAME = "tii_zcu111"
+ADDRESS = "192.168.0.81"
+PORT = 6000
+RUNCARD = pathlib.Path(__file__).parent / "tii_zcu111.yml"
+
+LO_ADDRESS = "192.168.0.212"
+
+
+def create(runcard=RUNCARD):
+    """Platform for ZCU111 board running qibosoq.
+
+    IPs and other instrument related parameters are hardcoded in.
+    """
+    # Create channel objects
+    channels = ChannelMap()
+
+    # Readout channel
+    channels |= ChannelMap.from_names("L3-30_ro")  # dac6
+
+    # QUBIT 0
+    channels |= ChannelMap.from_names("L2-4-RO_0")  # feedback adc0
+    channels |= ChannelMap.from_names("L4-29_qd")  # drive    dac3
+    channels |= ChannelMap.from_names("L1-22_fl")  # flux     dac0
+    # QUBIT 1
+    channels |= ChannelMap.from_names("L2-4-RO_1")  # feedback adc1
+    channels |= ChannelMap.from_names("L4-30_qd")  # drive    dac4
+    channels |= ChannelMap.from_names("L1-23_fl")  # flux     dac1
+    # QUBIT 2
+    channels |= ChannelMap.from_names("L2-4-RO_2")  # feedback adc2
+    channels |= ChannelMap.from_names("L4-31_qd")  # drive    dac5
+    channels |= ChannelMap.from_names("L1-24_fl")  # flux     dac2
+
+    # Map controllers to qubit channels (HARDCODED)
+    channels["L3-30_ro"].ports = [("dac6", 6)]
+
+    # Qubit 0
+    channels["L2-4-RO_0"].ports = [("adc0", 0)]
+    channels["L4-29_qd"].ports = [("dac3", 3)]
+    channels["L1-22_fl"].ports = [("dac0", 0)]
+    # Qubit 1
+    channels["L2-4-RO_1"].ports = [("adc1", 1)]
+    channels["L4-30_qd"].ports = [("dac4", 4)]
+    channels["L1-23_fl"].ports = [("dac1", 1)]
+    # Qubit 2
+    channels["L2-4-RO_2"].ports = [("adc2", 2)]
+    channels["L4-31_qd"].ports = [("dac5", 5)]
+    channels["L1-24_fl"].ports = [("dac2", 2)]
```

Comment:
In principle, you could write it only once, and iterate. Second and third qubits can be obtained from the first one, increasing some suitable variables.

However, they are only three. Do as you prefer.

Reply:
Being just three qubits, I like to have them explicit :-)